wiki/Installation/Linux/en.md

```diff
@@ -0,0 +1,351 @@
+```
+
+9. osu! will be installed under your `~/.wine/drive_c/Program Files/osu!` folder.
+
+10. You can now enjoy playing osu! on Linux!
+
+### Starting osu!
+
+To play osu! after you have closed it, you can:
+
+-   Navigate to the `~/.wine/drive_c/Program Files/osu!` folder and double click `osu!.exe`
+
+-   Create a shortcut to `~/.wine/drive_c/Program Files/osu!/osu!.exe`, if your desktop environment allows it.
+
+-   Run the following command:
+```
+$ WINEPREFIX=~/.wine wine '~/.wine/drive_c/Program Files/osu!/osu!.exe'
+```
+
+-   Create a [desktop entry](https://wiki.archlinux.org/index.php/desktop_entries) (this creates an application icon to click on, if your desktop environment does this).
+```
+$ cat > ~/.local/share/applications/osu!.desktop << "EOF"
+[Desktop Entry]
+Type=Application
+Name=osu!
+MimeType=application/x-winexe;
+Exec=env WINEPREFIX="~/.wine" wine "~/.wine/drive_c/Program Files/osu!/osu!.exe"
+Comment=Rhythm is just a click away!
+Categories=Game
+Icon=583A_osu!.0
+EOF
+```
+
+-   Create a script. If this id done, you can run `osu` in the terminal to start osu!
+```
+$ cat > osu << "EOF"
+#!/bin/sh
+
+export vblank_mode=0
+export WINEARCH=win32
+export WINEPREFIX=$HOME/osu-wine
+wine ~/osufolder/osu\!.exe "$@"
+EOF
+
+$ sudo mv osu /usr/bin/
+$ sudo chmod +x /usr/bin/osu
+```
+
+### Creating a kill script
+
+There are times that osu! may freeze or may not start up at all. If that does happen, you will need to kill the wineserver for your Wine prefix using:
+
+```
+$ WINEPREFIX=~/.wine wineserver -k
+```
+
+---
+
+You could also create a short script to do this quickly for you, as follows:
+
+```
+# cat > osukill << "EOF"
+#!/bin/sh
+
+export WINEPREFIX=~/.wine
+wineserver -k
+EOF
+
+$ sudo mv osukill /usr/bin/
+$ sudo chmod +x /usr/bin/osukill
+```
+
+Once this is done, you can run `osukill` in the terminal to force-kill osu!
+
+## Tweaks
+
+### Reducing sound latency
+
+You can set the sound driver to use alsa to help reduce latency.
+```
+$ WINPREFIX=~/.wine winetricks sound=alsa
+```
+
+The above should work with PulseAudio. If not, revert back to pulseaudio.
+```
+$ WINPREFIX=~/.wine winetricks sound=pulse
+```
+
+#### Registry tweak
+
+You can also do a registry tweak to further minimize sound latency.
+
+```
+$ cat > dsound.reg << "EOF"
+Windows Registry Editor Version 5.00
+
+[HKEY_CURRENT_USER\Software\Wine\DirectSound]
+"HelBuflen"="512"
+"SndQueueMax"="3"
+EOF
+
+$ WINEPREFIX=~/.wine wine regedit dsound.reg
+```
+
+This will generate a file call `dsound.reg`, once you run it with Wine, you can delete it or run the following:
+
+```
+$ rm desound.reg
+```
+
+### Font hinting issues
+
+To fix this, run the following:
+
+```
+WINEPREFIX=~/.wine winetricks settings fontsmooth=rgb
+```
+
+### Other notes
+
+#### Full screen mode
+
+If your desktop enviroment has special compositor and effects, you may want to keep osu! in fullscreen mode. As it will bypass the compositor.
```

Comment:
typo'd environment

change the period for a comma after "fullscreen mode" (and uncapitalize "As")